package.json

```diff
@@ -21,6 +21,8 @@
     "prepublish": "stencil build --prod",
     "firebase:deploy": "node scripts/deploy.js",
     "firebase:deploy:dev": "firebase use default && firebase deploy --token \"$FIREBASE_TOKEN\"",
+    "firebase:deploy:qa": "firebase use qa && firebase deploy --token \"$FIREBASE_TOKEN\"",
+    "firebase:deploy:staging": "firebase use staging && firebase deploy --token \"$FIREBASE_TOKEN\"",
```

Comment:
Do we still need these scripts here? We are no longer calling them in deploy.js. Maybe we still need them for manual deployments?

Reply:
yes, I am going to leave them in case they are needed for manual deployment.

Reply:
Ok sounds good